### PR TITLE
T1739 fix language selection

### LINKED ISSUE #1739: Language selection not working
Steps to reproduce in stage:
1. Search on "dictionarium".
2. Select edit for Dictionarium bibliothecarr...
3. For Main Title, click Language: English.

Expected result:
You are changing language for Main Title.

Actual result:
You are changing result for a different field.

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #1739

### CODE REVIEW

File: __tests__/components/ModalWrapper.test.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
-import { render } from '@testing-library/react'
+import { cleanup, render, prettyDOM } from '@testing-library/react'
```
Comment (justinlittman) [inline __tests__/components/ModalWrapper.test.js]: prettyDOM isn't used, so wondering why lint didn't catch.

**Comment by justinlittman** [comment]
For me locally:
```
/Users/jlittman/data/ld4p/sinopia_editor/__tests__/components/ModalWrapper.test.js
  4:27  warning  'prettyDOM' is defined but never used  no-unused-vars

/Users/jlittman/data/ld4p/sinopia_editor/src/components/ModalWrapper.jsx
  23:43  warning  The ref value 'el.current' will likely have changed by the time this effect cleanup function runs. If this ref points to a node rendered by React, copy 'el.current' to a variable inside the effect, and use that variable in the cleanup function  react-hooks/exhaustive-deps
  24:6   warning  React Hook useEffect has a missing dependency: 'modalRoot'. Either include it or remove the dependency array                                                                                                                                         react-hooks/exhaustive-deps
```

File: __tests__/components/ModalWrapper.test.js (hunk 1 above)
Comment (jermnelson) [reply __tests__/components/ModalWrapper.test.js]: Removed

**Comment by jermnelson** [comment]
@justinlittman eslint now passing. I'm also concerned that the the circle-ci didn't fial. Looking in the circle-ci job, the linter did return the three warnings but the step itself did not fail.

File: __tests__/components/ModalWrapper.test.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
-import { render } from '@testing-library/react'
+import { cleanup, render } from '@testing-library/react'
+import { setupModal } from '../test_utilities/testUtils'
```
Comment (justinlittman) [inline __tests__/components/ModalWrapper.test.js]: This should just be: `import { setupModal } from 'testUtils'`. I'm going to fix.

**Pull request merged**